R/estimate_infections.R

```diff
@@ -18,8 +18,9 @@
 #' estimate Rt for Covid-19 in a country from the ECDC data source.
 #'
 #' @param data A `<data.frame>` of disease reports (confirm) by date (date).
-#' `confirm` must be numeric and `date` must be in date format. Optionally
-#' this can also have a logical `accumulate` column which indicates whether
+#' `confirm` must be numeric and `date` must be in date format. `confirm` can
+#' also be set to a series of `NA` to do prior predictive checks. Optionally,
+#' `data` can also have a logical `accumulate` column which indicates whether
```

Comment:
I think this is less important than, say, the accumulation feature so I'd move it to the end. Slight rephrase suggested (as one might want to do prior predictions for other reason than checks).
```suggestion
#' `confirm` must be numeric and `date` must be in date format. If all entries of
#' `confirm` are missing (`NA`) the returned estimates will represent the prior distributions. Optionally, 
#' `data` can also have a logical `accumulate` column which indicates whether
```

Reply:
This comment?  Sorry I thought you had already implemented what you meant. 

My thinking is that it makes sense for it follow on from what is required of ``date` and `confirm`. I don't read it to mean it's more important. 

If still going by your suggestion, what do you mean by the "end"? The end of the whole text block?